### ISSUE TITLE
Prefab and meta serialization is not up to date

### ISSUE THREAD

**Issue by gilbdev** [microsoft/MixedRealityToolkit-Unity#4914]
## Describe the bug
prefab, meta, and scene assets are not automatically updated in Unity.
the serialization of those files can change when:
- unity version is updated
- some code of the components used in prefab has changed on the serialization part (renaming adding,removing serialized field)

currently, if i make a change in some of the mrtk prefabs, a whole list of diffs will come up.

bonus effect: if you make a tool (see below) in the mrtk, this can be useful when people are upgrading version on their side (to reserialize their profiles and scenes for example)

## Expected behavior

when a new mrtk release is pushed, all files should be up to date regarding serialization, at least the changes regarding mrtk code. (untiy serialization changes is a subject to be discussed, i don't know the exact repercussion for retrocompatibilities).

##
this process can be automated:
(i can provide a better version that displays a progress bar)
```

[MenuItem("Assets/Reserialize Prefabs&Scenes...")]
private static void ReserializeAll()
{
	var array = GetAssets("t:Prefab t:Scene t:ScriptableObject");
	AssetDatabase.ForceReserializeAssets(array);
}
private static string[] GetAssets(string filter)
{
	string[] allPrefabsGUID = AssetDatabase.FindAssets($"{filter}");

	List<string> allPrefabs = new List<string>();
	foreach (string guid in allPrefabsGUID)
	{
		allPrefabs.Add(AssetDatabase.GUIDToAssetPath(guid));
	}
	return allPrefabs.ToArray();
}
```

**Comment by gilbdev** [comment]
reserializing assets also helps pointing out 'missing scripts' like #4738 
(when running my reserialize script, i got an error :
`You are trying to replace or create a Prefab from the instance 'ToggleFeaturesPanel' that references a missing script. This is not allowed.` )

**Comment by gilbdev** [comment]
fixed in v2.0.0
